homework/12_B/03_Andrey/homework2/chess_board.py

```diff
@@ -0,0 +1,35 @@
+import termcolor
```

Comment:
I really like the way you print out the board. But can you also print the cell colors i.e. alternating black and white

Reply:
I don't think it is possible to add background color without coloring outside of the board with the current border system